index.html

```diff
@@ -8,16 +8,19 @@
   <title>My portfolio</title>
   <link rel="stylesheet" href="style.css">
 </head>
-
 <body>
   <header class="header">
     <a href="#" class="logo">my logo</a>
```

Comment:
- [ ] Please add blur effect and decrease the color transparency for the mobile menu background and make sure it matches the design on the template.
  -To add blur you can use this `backdrop-filter: blur(Npx);`

### Your design

![image](https://user-images.githubusercontent.com/83803414/161629171-245dc846-5c55-4017-85b2-5fca0b61b77d.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/161629218-dbe35181-4e5f-4508-8038-6fe07636b06a.png)

Reply:
Addressed ✔